action.yml

```diff
@@ -400,7 +400,7 @@ runs:
         INPUT_DRIFT_DETECTION_SLACK_NOTIFICATION_URL: ${{ inputs.drift-detection-slack-notification-url }}
         NO_BACKEND: ${{ inputs.no-backend }}
         DEBUG: 'true'
-        TERRAGRUNT_PROVIDER_CACHE: ${{ inputs.cache-dependencies == 'true' && 1 || 0 }}
+        TG_PROVIDER_CACHE: ${{ inputs.cache-dependencies == 'true' && 1 || 0 }}
```

Comment:
we need both TERRAGRUNT_PROVIDER_CACHE_DIR and TG_PROVIDER_CACHE_DIR here to be backwards compatible with older terragrunt versions